packages/ai-chat/src/common/architect-task-summary-agent-prompt.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/tslint/config  */
+// *****************************************************************************
+// Copyright (C) 2025 EclipseSource GmbH and others.
+//
+// This file is licensed under the MIT License.
+// See LICENSE-MIT.txt in the project root for license information.
+// https://opensource.org/license/mit.
+//
+// SPDX-License-Identifier: MIT
+
+export const ARCHITECT_TASK_SUMMARY_PROMPT = {
```

Comment:
Should there be an actual prompt defined here?

Reply:
no, this is not used anymore. This will be removed in the follow up commit